content/posts/2020/2020-09-09-bicep-sneak-peek.md

```diff
@@ -13,9 +13,9 @@ cover: https://sa0blogs.blob.core.windows.net/devkimchi/2020/09/bicep-sneak-peek
 fullscreen: true
 ---
 
-There are several ways provisioning resources onto [Azure][az]. One of those approaches is to use the [ARM template][az arm template]. However, as many DevOps engineers have been providing feedback that ARM template is hard to learn and deploy at scales, as it can be tricky. Therefore, field experts like Microsoft MVPs have suggested many best practices about authoring ARM templates, but it's still the big hurdle to get through.
+Today there are several different ways of provisioning resources onto [Azure][az] either via the Azure Portal or in a programmatic way using PowerShell, Azure CLI or Azure SDKs available for several different languages, which in the end all leverages Azure's REST-based Resource Manager (ARM). One of those programmatic approaches is to use the JSON-based [ARM templates][az arm template]. However, many customers and especially DevOps engineers have been providing feedback that ARM templates are hard to learn, modify and deploy at scale. Therefore, field experts like Microsoft MVPs have suggested many best practices about authoring ARM templates, as well as the community has created a large set of samples in the Azure QuickStart Templates Library on GitHub, but it's still the big hurdle to get through.
```

Comment:
I think choose either several or different not both. 

Last sentence recommended change:

The community has created recommended practices in authoring ARM templates and contributed samples to the Azure QuickStart Templates Library on GitHub to help reduce the challenges with working directly with ARM templates. To complement these efforts, Microsoft is working to improve the overall experience of defining infrastructure as code using the new Bicep ARM Template DSL.